webpack.config.js

```diff
@@ -1,11 +1,22 @@
 const path = require('path');
 
-module.exports = [
-    buildJsFromTo({from: 'plugins/plugin.archive_analytics.js', to: 'BookReader/plugins/plugin.archive_analytics.js'}),
-    buildJsFromTo({from: 'plugins/menu_toggle/plugin.menu_toggle.js', to: 'BookReader/plugins/plugin.menu_toggle.js'}),
-    buildJsFromTo({from: 'plugins/tts/plugin.tts.js', to: 'BookReader/plugins/plugin.tts.js'}),
+const listOfFiles = [
+    'plugins/plugin.archive_analytics.js',
+    'plugins/menu_toggle/plugin.menu_toggle.js',
+    'plugins/tts/plugin.tts.js',
 ];
 
+const compileFiles = async () => {
+    const nestedDirs = new RegExp('/(.*)/');
+    return await listOfFiles.map((filePath) => {
+        const flattenedFilePath = filePath.replace(nestedDirs, '/');
```

Comment:
++; this is a really perfect example of a good variable name. Although we could inline this since it's only used once, I think that would result in the code being harder to understand.